force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,90 @@
+<template>    
+    <lightning-card  title="PD1 Calculator"> 
+        <!-- Process -->
+        <lightning-button 
+            variant="brand" 
+            label="Calculate" 
+            title="Calculate" 
+            slot="actions" 
+            onclick={calculateScore}
+        ></lightning-button>
```

Comment:
Very advanced but try to remove this button and if the values change it automatically calculates.